include/dbstructure.php

```diff
@@ -176,7 +186,7 @@ function update_structure($verbose, $action, $tables=null, $definition=null) {
 				if ($current_index_definition != $new_index_definition && substr($indexname, 0, 6) != 'local_') {
 					$sql2=db_drop_index($indexname);
 					if ($sql3 == "")
-						$sql3 = "ALTER TABLE `".$name."` ".$sql2;
+						$sql3 = "ALTER IGNORE TABLE `".$name."` ".$sql2;
```

Comment:
See my comment concerning MySQL 5.7 which doesn't support this statement anymore. Either we check the version number here or we do it differently.